tutorials/aperiodic/plot_Fluctuations.py

```diff
@@ -0,0 +1,153 @@
+"""
+Fluctuation analyses
+====================
+
+Apply fluctuation analyses, such as detrended fluctuation analysis (DFA) to neural signals.
+
+DFA was first proposed in the context of genetics in
+`Peng et al, 1994 <https://doi.org/10.1103/PhysRevE.49.1685>`_,
+and was recently reviewed in the context of neural data in
+`Hardstone et al, 2012 <https://doi.org/10.3389/fphys.2012.00450>`_.
+
+This tutorial covers ``neurodsp.aperiodic.dfa``.
+"""
+
+###################################################################################################
+
+# sphinx_gallery_thumbnail_number = 1
+
+import matplotlib.pyplot as plt
+
+from neurodsp.sim import sim_powerlaw
+
+# Import the function for computing fluctuation analyses
+from neurodsp.aperiodic import compute_fluctuations
+
+###################################################################################################
+# Detrended Fluctuation Analysis
+# ------------------------------
+#
+# Detrended fluctuation analysis is a method for analyzing the self-similarity of a signal.
+#
+# It is in some ways similar to looking at autocorrelation, and can is typically used to look
+# for long-range, powerlaw correlations. It does so by dividing the signals into windows,
+# fitting local trends, and then examining the pattern across window sizes.
+#
+# For more information on the details of DFA, see the description of the algorithm on
+# `Wikipedia <https://en.wikipedia.org/wiki/Detrended_fluctuation_analysis>`_.
+#
+# DFA Exponent
+# ~~~~~~~~~~~~
+#
+# The output of the DFA algorithm is the 'DFA exponent', which is the slope of the linear fit
+# between windows sizes and fluctuations. The value of this exponent can be interpreted in
+# terms of the properties of the signal.
+#
+# In particular, DFA exponents represent:
+#
+# - :math:`0 < \alpha < 0.5` : anti-correlated
+# - :math:`\alpha = 0.5` : white noise
+# - :math:`0.5 < \alpha < 1` : correlated (stationary)
+# - :math:`\alpha = 1.0` : 1/f
+# - :math:`1 < \alpha < 2` : correlated (non-stationary)
+#
+
+###################################################################################################
+# Applying DFA
+# ------------
+#
+# Here, to introduce DFA, we will use colored noise signals, (white noise and pink noise),
+# with have different auto-correlation properties, and so should show different DFA results,
+# which we can compare.
+#
+# Note that DFA can be applied to multiple signal types, and in neural field data it is
+# often used to examine amplitude time series of neural oscillations.
+#
+
+###################################################################################################
+
+# Simulation settings settings
```

Comment:
first bursts bursts and now settings settings :wink:

Reply:
oh no. word are hard :(